src/Compilers/Core/Portable/Operations/ICatchClause.cs

```diff
@@ -11,7 +11,7 @@ namespace Microsoft.CodeAnalysis.Semantics
     /// This interface is reserved for implementation by its associated APIs. We reserve the right to
     /// change it in the future.
     /// </remarks>
-    public interface ICatchClause : IOperation
+    internal interface ICatchClause : IOperation // https://github.com/dotnet/roslyn/issues/22008
```

Comment:
>ICatchClause [](start = 23, length = 12)

It doesn't look like we have a single test covering this for C#.

Reply:
Same for VB.

---
In reply to: [139036832](https://github.com/dotnet/roslyn/pull/22018#discussion_r139036832) [](ancestors = 139036832)